db/seeds.rb

```diff
@@ -549,6 +549,51 @@ def require_updates?(real_attrs, seed_attrs)
   is_project: true,
   url: "/web_development_101/project_rails.md"
 )
+# +++++++++++
+# SECTION
+# +++++++++++
+
+section_position += 1
+section = create_or_update_section(
+  title: "Pair Programming",
```

Comment:
@KevinMulhern I can understand your concern, but I think that marking the whole section optional is not the best idea:

1. Pair Programming is a common practice in the real world that it is important students learn about, even if they don't want to do it as part of TOP.
2. The First two lessons do not actually require any pairing--that isn't till the project, so there is no reason to make them optional
3. Making it optional on the front will tend to make it harder to find pairs in the chat room as less people overall will bother with the section (thus making it a self-fulfilling prophecy)
4. I know we don't want to make the bar too high, but we don't want it too low that we don't expect the students to do things that they are going to want to have knowledge and experience of to improve their chances of getting a job--that'd be doing them a disservice.

All that aside, you do have a point in that some people might have social anxiety or other legitimate issues--as well as early on there may be less students looking for pairs. To that end, rather than marking optional on the top, I'd propose modifying the project description to emphasize the "We'd really like you do do this pair project for the experience, but if you don't want to, or can't, just make sure you understand the first two lessons." Or something along those lines.